### DISCUSSION TITLE
Novos casos classificados por estado de vacinação?

### DISCUSSION THREAD

**Discussion by pma2** [dssg-pt/covid19pt-data#1037] [Geral]
Boas :) Parabéns pelo repo, está espetacular! Fiquei maravilhado de saber que esta iniciativa existe :)

Dei uma vista de olhos ainda superficial nas fontes de dados, e não encontrei informação sobre isto:

- classificação dos novos casos de covid por estado de vacinação, exemplo: não vacinado, 1 dose, 2 doses, ...
- Idem para casos internados, casos UCI e mortes.
- Extras interessantes seriam o tipo de vacina, há quanto tempo foi vacinado, e a gravidade dos sintomas. E "já agora" se teve covid antes.

Penso que esta informação deve existir na DGS, não sei se é pública, mas seria muito interessante para analizar o efeito das vacinas.

Têm conhecimento se este tipo de informação está disponivel, e em caso afirmativo, se pode ser integrada nas fontes neste repo?

Algo semelhante ao que se publica para o UK aqui https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1005517/Technical_Briefing_19.pdf (ver pág 18). Ter informação da variante imagino que seja difícil atualmente, mas pelo menos informação sobre o estado de vacinação dos novos casos Covid seria muito interessante.

Grato!

**Comment by davipt** [comment]
Esta informação, caso exista, não acredito que seja disponibilizada ao público de forma anónima na Internet, visto que a informação detalhada (mas anonimizado) existente de internados e datas de sintomas e muito mais, usada por investigadores e pelo INSA para o cálculo do Rt, está disponíveis apenas após preenchimento do formulário em https://covid19.min-saude.pt/disponibilizacao-de-dados/ No entanto pelo que vejo na descrição dos metadados, não foi ainda estendido para incluir informação de vacinação.

**Reply by pma2** [reply]
Grato pela resposta David, faz sentido.

É pena que não exista mais transparência e disponibilização de acesso aos dados ao público pela DGS / outras autoridades. Acho que a sociedade só tem a ganhar, não vejo nenhuma vantagem em criar barreiras de acesso a este tipo de informação, eventualmente agregada. Assim temos apenas as referências dos resultados dos países onde esses dados são divulgados, mas seria interessante ver dados para Portugal.

**Reply by davipt** [reply]
Como disse, os dados estão disponíveis (vacinação suspeito que não), mas apenas a quem se inscreva e justifique o acesso aos dados.

Informação pontual e sumarizado, como no PDF acima, estará certamente disponível por exemplo nos relatórios das reuniões do Infarmed. Mas como essa informação não é estruturada, não tem sido incluido aqui no repositório. Por exemplo importamos os dados de Rt, mas não o relatório em PDF. Mas para as vacinas ficamos com uma cópia do relatório PDF, por redundância.

Conheces países com informação detalhada e estruturada, para alem do PDF acima? Havendo exemplos de outros países, ou quiçá até de países suficientes para estar integrado no OWID, seria mais fácil fazer pressão cá dentro para haver os mesmos dados.
